akka-ssl-config/src/main/scala/com/typesafe/sslconfig/akka/AkkaSSLConfig.scala

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package com.typesafe.sslconfig.akka
```

Comment:
Yes, it's the same module (and should become a replacement of the old one) so we should reconsider if renaming could be a problem.

Assuming that no one references this module manually, it probably wouldn't be a problem, because akka-stream will just point to the new module. Otherwise, if people  already depend on the old module explicitly, they might end up with to versions on their classpath if they also use akka-stream in the new version now depending on the new module.

So, keeping the old artifact coordinates could make sense. In this case, the module would need to be special-cased to also use the previous maven organization.